tensorflow/lite/micro/benchmarks/micro_benchmark.h

```diff
@@ -77,6 +77,10 @@ class MicroBenchmarkRunner {
     interpreter_.GetMicroAllocator().PrintAllocations();
   }
 
+  tflite::RecordingMicroInterpreter* GetInterpreter() {
```

Comment:
nit: let's return a mutable reference. That way we know it can not be nullptr.

http://go/totw/labs/return-by-pointer-or-reference#discouraged-alternative

Reply:
Done.